docs/vwii/vwii-modding.md

```diff
@@ -29,70 +27,53 @@ We will now place the required Homebrew files on the SD Card.
 1. Copy the `compat_installer.elf` file to the `wiiu/apps` folder on the root of your SD Card.
 1. Take the SD Card out of your computer and plug it into your Wii U console.
 
-### NAND Backup
-
-In case anything goes wrong in the later process and your vWii ends up bricked, restoring a previously made NAND backup can fix it.
-
-?> If you have recently made a NAND backup that includes SLCCMPT and OTP, feel free to skip this step.
-
-1. Launch the [Wii U NAND Dumper](vwii/browser-exploit).
-1. Use the Wii U GamePad's D-Pad to enter the following configuration:
-    - Dump SLC: **optional**
-    - Dump SLCCMPT: **yes**
-    - Dump MLC: **optional**
-    - Dump OTP: **yes**
-    - Dump SEEPROM: **optional**
-1. Press the A button to start the dumping process.
-1. When the process completed, power off your Wii U, take your SD Card out of the Wii U and plug it into your PC.
-1. To make sure you don't lose the files, copy the `slccmpt.bin`, `otp.bin` and if you chose to go with a full backup, `seeprom.bin`, `slc.bin`, and `every mlc.bin.part` file to your computer.
-1. Delete the files from your SD Card to free up space.
-1. Take the SD Card out of your computer and plug it into your Wii U console.
+?> Please ensure youve made a NAND Backup before continuing. If you have not, please go to [here]( https://wiiu.hacks.guide/#/tiramisu/nand-backup) for instructions.
 
 ### Installing the Homebrew Channel
 
-1. Boot into [Tiramisu](vwii/browser-exploit).
-1. Launch the Homebrew Launcher, by opening the Mii Maker.
-1. Launch the compat_installer.
-1. Press `A` to install the Homebrew Channel and wait until you see `Install succeeded`. Then press the HOME button to return to the Wii U Menu.
-1. Launch vWii (the Wii Menu icon).
-   - If the installation has succeeded, you should see the Homebrew Channel in your Wii Menu.
+1. Boot into Aroma
+1. Launch the Homebrew Launcher, by opening the Mii Maker

```

Comment:
Not sure how you can boot into Aroma and then launch HBL when HBL does not exist on Aroma